main.c

```diff
@@ -490,6 +494,9 @@ static void destroy_output(struct slurp_output *output) {
 		wl_callback_destroy(output->frame_callback);
 	}
 	wl_output_destroy(output->wl_output);
+	if (output->logical_geometry.label) {
```

Comment:
No need for this NULL check -- `free(NULL)` is a no-op.

Reply:
oh, nice.